README.md

```diff
@@ -38,14 +38,18 @@ repositories {
 }
 
 dependencies {
-    implementation "de.smartsquare:socket-io-redis-emitter:0.11.3"
+    implementation "de.smartsquare:socket-io-redis-emitter:0.11.3"   
 }
 ```
 
+This library comes with a default implementation for both jedis and lettuce.
+
 ### Emit Cheatsheet
 
+When using Jedis:
 ```kotlin
-val emitter = Emitter(JedisPool("localhost"), namespace = "/")
+val jedisConnection = jedisConnectionFactory.connection as JedisConnection
+val emitter = Emitter(JedisPublisher(jedisConnection.jedis))
```

Comment:
Is this Spring specific code? I don't think this belongs in the README (or would need a specific section) since this is a package without dependency on Spring.

Reply:
I added a dedicated section for this.